File: ex05/ex05_02/ex05_02.py
def main():
    number = 0
    print("Enter a number to compute its factorial: ")
    number = int(input())
    factorial = 1
    for i in range(1, number + 1):
        factorial *= i

    print("The factorial of", number, "is", factorial)
    return 0

if __name__ == "__main__":
    main()  

File: ex05/ex05_02/ex05_02.c
#include <stdio.h>

int main() {
    int number = 0;
    printf("Enter a number to compute its factorial: ");
    scanf("%d", &number);

    long factorial = 1;
    for (int i = 1; i <= number; i++) {
        factorial *= i;
    }
    printf("The factorial of %d is %ld.\n", number, factorial);
    return 0;
}